tests/test_comment.py

```diff
@@ -0,0 +1,12 @@
+"""This module test the counter.py module regarding comments"""
+from castanet import counter
+from castanet import generate_trees as generator
+
+def test_comment():
```

Comment:
Please have your tests use a different directory other than `hello`. You can create these files in the `tests` folder.